### PR TITLE
Remove columns argument

### LINKED ISSUE #73: Documentation for `columns` argument in `check.survey()` is not accurate
The documentation for `columns` states:

> if given, a named character vector containing the name of the "id", "participant.age" and "contact.age" columns

But the value of this argument is never used in the code. We simply check is a value, any value, is passed.

Since the case where no value is passed is not very interesting (it's just testing if the object is a `data.frame` or a `matrix`), I would recommend removing this argument and consider that it's always set.

### PR DESCRIPTION
**Pull request by sbfnk**
Fixes #73

### CODE REVIEW

**Comment by sbfnk** [comment]
@Bisaloo feel free to have a look as it was you who kindly raised the Issue but otherwise fairly confident that it's OK to just go ahead and merge.

**Review by Bisaloo — COMMENTED** [review]
Thanks!

Could you please:

- [x] Edit the documentation to highlight that this argument is deprecated.

I also want to take a moment to think about argument order. I don't know if this argument should stay in this position, as it would imply that all the following arguments have to be named. OTOH, if we move it and someone specified it by position in an old script, it will erroneously be passed to `id.column`.

**Comment by sbfnk** [comment]
> I also want to take a moment to think about argument order. I don't know if this argument should stay in this position, as it would imply that all the following arguments have to be named. OTOH, if we move it and someone specified it by position in an old script, it will erroneously be passed to id.column.

Tricky one. Perhaps keep where it is for now (not breaking existing codes) and remove in 0.4.0 given that people have been warned?

**Comment by sbfnk** [comment]
@Bisaloo any further thoughts? If not I’ll go ahead with this:

> Perhaps keep where it is for now (not breaking existing codes) and remove in 0.4.0 given that people have been warned?

File: R/check.r [hunk 1]
```diff
@@ -7,7 +7,7 @@ check <- function(x, ...) UseMethod("check")
 #' @description Checks that a survey fulfills all the requirements to work with the 'contact_matrix' function
 #'
 #' @param x A [survey()] object
-#' @param columns if given, a named character vector containing the name of the "id", "participant.age" and "contact.age" columns
+#' @param columns depcreated argument, ignored
```
Comment (Bisaloo) [inline R/check.r:10]:
```suggestion
#' @param columns deprecated argument, ignored
```

**Comment by Bisaloo** [comment]
> Tricky one. Perhaps keep where it is for now (not breaking existing codes) and remove in 0.4.0 given that people have been warned?

Yes, seems like the least worse option. It'll be slightly more annoying for users but cannot generate any unexpected behaviour.

Just a note that a strict semantic versioning adherence would probably warrant a jump to 1.0.0 to remove this argument.

**Pull request merged**